src/api.service.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+import { AssetsResponse, GetAssetsParams } from './interfaces/assets.interface';
+
+export const BASE_URL = 'https://api.better-call.dev/v1';
+export const api = axios.create({ baseURL: BASE_URL });
+
+export async function requestGetAssets(params: GetAssetsParams): Promise<AssetsResponse> {
```

Comment:
Incorrect english word order. 
Rename to `getAssetsRequest` or just `getAssets`